README.rst

```diff
@@ -0,0 +1,49 @@
+===================
+RDO Project Easyfix
+===================
+
+It is a collection of easyfix or low hanging fruits issues for RDO Project.
+
+I am new to RDO Project, How can i contribute?
+==============================================
+
+* You can check the issues and assign to it yourself.
+* Each issue contains mentor IRC nick.
+* Read the issue carefully and go through the links attached to issue.
+* If you are not able understand, join the IRC channel #rdo on Freenode server.
+* Ping the Mentor regarding the issue, S/He will help to move forward.
+
+I want to add easyfix, How can i do that?
+=========================================
+* Go to issues and file the issue.
```

Comment:
[Issues](https://github.com/redhat-openstack/easyfix/issues) link for convenience